source/isaaclab_tasks/isaaclab_tasks/manager_based/classic/humanoid/agents/sb3_ppo_cfg.yaml

```diff
@@ -14,9 +14,9 @@ n_epochs: 5
 gae_lambda: 0.95
 max_grad_norm: 1.0
 vf_coef: 0.5
-policy_kwargs: "dict(
-  activation_fn=nn.ELU,
-  net_arch=[400, 200, 100],
-  optimizer_kwargs=dict(eps=1e-8),
-  ortho_init=False,
-  )"
+policy_kwargs:
+  activation_fn: 'nn.ELU'
```

Comment:
What happens if user does `torch.nn.ELU`? Is this something SB3 doesn't support? Why not instead check if there is "_fn" argument and parse the string for it?